src/lib/api/schema/atoms/Age.schema.ts

```diff
@@ -1,3 +1,15 @@
 import { z } from 'zod';
 
-export const AgeSchema = z.coerce.string(); // Both number and string will work
+const MIN_AGE = 6;
+const MAX_AGE = 100;
```

Comment:
Not sure if this is correct - it looks like there is no max age as per API docs "age": "<number = 6 - 100+ ",

Reply:
I guess this is a theoretical problem as in practice I would be surprised if physicians work when they're after 100yo 😛

Reply:
If we are adding validation to match their validation then we should do it 1:1 plus I'm still not 100% sold whether it's the patient's age or the physician's age. I will update it tomorrow.